extensions/container-image/container-image-docker/deployment/src/main/java/io/quarkus/container/image/docker/deployment/DockerConfig.java

```diff
@@ -32,6 +33,7 @@ interface DockerBuildxConfig {
          * Which platform(s) to target during the build. See
          * https://docs.docker.com/engine/reference/commandline/buildx_build/#platform
          */
+        @WithDefault("local")
         Optional<List<String>> platform();
```

Comment:
Ideally, please remove the `Optional` given we don't need it anymore. I would expect us to always want it set to something, right?

Reply:
Since, you have a default value, then the optional will always hold a value, so I am not sure why you would want it here. In other words I agree with @gsmet. It should be removed.